packit_service/worker/handlers/testing_farm.py

```diff
@@ -135,7 +135,7 @@ def _get_or_create_group(
                 trigger_id=self.db_trigger.id,
             )
             if self.testing_farm_job_helper.skip_build or not copr_build
-            else copr_build.runs[-1]
+            else copr_build.group_of_targets.runs[-1]
```

Comment:
I love how Python has a lot of useless methods, yet you cannot do `.last()` like a civilized person…

---

I might've been corrupted by either Kotlin or Rust…